src/components/icon/demoLoadSvgIconsFromUrl/index.html

```diff
@@ -1,4 +1,4 @@
-<div ng-controller="DemoCtrl" ng-cloak layout="column" layout-margin  ng-cloak>
+<div ng-controller="DemoCtrl" ng-cloak layout="column" layout-margin>
```

Comment:
Why did you remove the `ng-cloak` which is used to hide flickers due to Layout API activity.